apps/docs/content/guides/platform/sentry-monitoring.mdx

```diff
@@ -0,0 +1,301 @@
+---
+title: 'Sentry integration'
+subtitle: 'Integrate Sentry to monitor errors from a Supabase client'
+---
+
+You can use [Sentry](https://sentry.io/welcome/) to monitor errors thrown from a Supabase JavaScript client. Install the [Supabase Sentry integration](https://github.com/supabase-community/sentry-integration-js) to get started.
+
+The Sentry integration supports browser, Node, and edge environments.
+
+## Installation
+
+Install the Sentry integration using your package manager:
+
+<Tabs scrollable queryGroup="package-manager" defaultActiveId="npm" type="underlined" size="small" >
+
+<TabPanel id="npm" label="npm">
+
+```sh
+npm install @supabase/sentry-js-integration
+```
+
+</TabPanel>
+
+<TabPanel id="yarn" label="yarn">
+
+```sh
+yarn add @supabase/sentry-js-integration
+```
+
+</TabPanel>
+
+<TabPanel id="pnpm" label="pnpm">
+
+```sh
+pnpm add @supabase/sentry-js-integration
+```
+
+</TabPanel>
+
+</Tabs>
+
+## Use
+
+To use the Supabase Sentry integration, add it to your `integrations` list when initializing your Sentry client.
+
+You can supply either the Supabase Client constructor or an already-initiated instance of a Supabase Client.
+
+<Tabs scrollable defaultActiveId="constructor" type="underlined" size="small">
+
+<TabPanel id="constructor" label="With constructor">
+
+```ts
+import * as Sentry from '@sentry/browser'
+import { SupabaseClient } from '@supabase/supabase-js'
+import { SupabaseIntegration } from '@supabase/sentry-js-integration'
+
+Sentry.init({
+  dsn: SENTRY_DSN,
+  integrations: [
+    new SupabaseIntegration(SupabaseClient, {
+      tracing: true,
+      breadcrumbs: true,
+      errors: true,
+    }),
+  ],
+})
+```
+
+</TabPanel>
+
+<TabPanel id="instance" label="With instance">
+
+```ts
+import * as Sentry from '@sentry/browser'
+import { createClient } from '@supabase/supabase-js'
+import { SupabaseIntegration } from '@supabase/sentry-js-integration'
+
+const supabaseClient = createClient(SUPABASE_URL, SUPABASE_KEY)
+
+Sentry.init({
+  dsn: SENTRY_DSN,
+  integrations: [
+    new SupabaseIntegration(supabaseClient, {
+      tracing: true,
+      breadcrumbs: true,
+      errors: true,
+    }),
+  ],
+})
+```
+
+</TabPanel>
+
+</Tabs>
+
+## Configuration
```

Comment:
I wonder if it's worth repeating all options here, as we'll need to remember to update two places now. Here and `README.md` of integration itself.

Reply:
I'm fine with moving it to `README.md`, but we'll need to add the param details for `payload` and `sanitizeBody`.

Took me a dive through the code to figure out where those are coming from, and we want to save readers the hassle

Reply:
Updated to use table + linked to TS types that include all the details https://github.com/supabase-community/sentry-integration-js/blob/master/README.md#options